spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/properties/ComposedTaskProperties.java

```diff
@@ -188,18 +188,10 @@ public class ComposedTaskProperties {
 	 */
 	private boolean splitThreadWaitForTasksToCompleteOnShutdown;
 
-	/**
-	 * Allows a single ComposedTaskRunner instance to be re-executed without
-	 * changing the parameters. It does this by applying a {@code run.id} with a sequential
-	 * number based on the {@code run.id} from the previous execution.   Default is true.
-	 */
-	@Deprecated
-	private boolean incrementInstanceEnabled = true;
```

Comment:
Do you mind starting a "What's New" doc and adding this as a "Removal"? This way when we release we already have all these breaking changes listed and don't have to backtrack and decipher what we removed. We could do something similar to [spring pulsar](https://github.com/spring-projects/spring-pulsar/blob/main/spring-pulsar-docs/src/main/antora/modules/ROOT/pages/whats-new.adoc). And really, I don't care if we just at a text file w/ a bullet point list of commits for now - just so we can easily find these removals later.

Wdyt?

Reply:
Agreed.   I'll create a new issue for it then create a PR for it.

Reply:
PR: https://github.com/spring-cloud/spring-cloud-dataflow/issues/6004